tests/Mocks/Servers.cpp

```diff
@@ -420,46 +444,52 @@ void MockClusterServer::startFeatures() {
   auto& factory =
       getFeature<arangodb::iresearch::IResearchFeature>().factory<arangodb::ClusterEngine>();
   indexFactory.emplace(arangodb::iresearch::DATA_SOURCE_TYPE.name(), factory);
+  _server.getFeature<arangodb::ClusterFeature>().clusterInfo().startSyncers();
```

Comment:
This looks like the right place to start the syncers, are the other two I saw wrong?

Reply:
The others are independent